pt-PT/Universally Preferable Behaviour/p02_ch03_the_second_test_murder.md

```diff
@@ -1,39 +1,39 @@
-## The Second Test: Murder
+## Segundo Teste: Assassinato
 
-Let us now test the UPB framework against moral propositions regarding *murder*, which here is defined as killing intentionally and with premeditation, not in self-defence.
+Deixe-nos testar a estrutura do UPB contra proposições morais sobre o *assassinato*, que aqui é definido como matar intencionalmente e com premeditação, não em legítima defesa.
 
-Since we spent so much time dissecting the question of rape – and since many of the same arguments will apply here – this analysis can be much briefer.
+Uma vez que passamos tanto tempo dissertando a questão da violação – e já que vários dos mesmos argumentos vão ser aplicados aqui – esta análise pode ser muito mais breve.
 
-Let us return to our two moral guinea pigs sitting in a room – we’ll call them Bob and Doug.
+Voltemos para as nossas duas cobaias morais sentados num quarto - iremos chama-los de Bob e Doug.
```

Comment:
the gender in "chama-los" is incorrect, since guinea pigs are translated into a feminine word

Reply:
You're right, as the names are male I made confusion. I'll change it.